test/unit/model_test.rb

```diff
@@ -1902,4 +1902,152 @@ class HasPaperTrailModelTransactionalTest < ActiveSupport::TestCase
       end
     end
   end
+
+  context "Models with nested has_many through relationships" do
+    setup { @chapter = Chapter.create(:name => 'ch_1') }
+
+    context "before any associations are created" do
+      setup do
+        @chapter.update_attributes(:name => "ch_2")
+        @ch_1 = @chapter.versions.last.reify(:has_many => true)
+      end
+
+      should "reify the record when reify is called" do
+        assert_equal "ch_1", @ch_1.name
+      end
+    end
+
+    context "after the first has_many through relationship is created" do
+      setup do
+        Timecop.travel 1.second.since
+        @chapter.update_attributes :name => "ch_2"
+        Timecop.travel 1.second.since
+        @chapter.sections.create :name => "section 1"
+        Timecop.travel 1.second.since
+        @chapter.sections.first.update_attributes :name => "section 2"
+        Timecop.travel 1.second.since
+        @chapter.update_attributes :name => "ch_3"
+        Timecop.travel 1.second.since
+        @chapter.sections.first.update_attributes :name => "section 3"
+      end
+
+      context "when reified is called" do
```

Comment:
The method is called `reify`, not `reified`.  Please update the context string here.